File: 2st/Yeonwoo/문자열 바꿔서 찾기.py
def solution(myString, pat):
    answer = 0
    myString_ = ''
    for i in myString:
        if i=='A':
            myString_ += i.replace('A','B')
        else:
            myString_ += i.replace('B', 'A')
    if pat in myString_:
        answer = 1
    return answer

File: 3st/Juram/programmer/문자열 바꿔서 찾기.js
function solution(myString, pat) {
    var answer = 0;
    var change = "";
    for(let i=0; i<myString.length; i++){
        if(myString[i]=="A") change += "B";
        else change += "A";
    }
    if(change.includes(pat)) return 1;
    else return 0;
}